lib/linguist/languages.yml

```diff
@@ -4227,6 +4227,11 @@ Smarty:
   codemirror_mime_type: text/x-smarty
   tm_scope: text.html.smarty
   language_id: 353
+Solidity:
+  type: programming
+  ace_mode: text
+  tm_scope: source.solidity
+  language_id: 237469032
```

Comment:
The [logo](https://solidity.readthedocs.io/en/latest/_images/logo.svg) only has dark shades of gray. I'm not sure it'll look very nice but I've set it to one of those.